lib/netext/httpext/tracer.go

```diff
@@ -184,7 +184,7 @@ func now() int64 {
 // Keep in mind that GetConn won't be called if a connection
 // is reused though, for example when there's a redirect.
 // If it's called, it will be called before all other hooks.
-func (t *Tracer) GetConn(hostPort string) {
+func (t *Tracer) GetConn(hostPort string) { //nolint:revive
```

Comment:
I was considering that, but found this unused params more human (documentation)-friendly that `_` + looking for a method description if somebody ever decided to start using them.

Reply:
All of these methods implement [this](https://pkg.go.dev/net/http/httptrace#ClientTrace) interface

That is good enough documentation in my opinion ... and there is very little chance that we will start using those arguments and the codebase will not change enough that this whole thing will not be rewritten.

So it seems like YAGNI situation to me.

On the other hand I would argue `nolint` usages should be more about cases where we *want* to keep it as there is a good reason. Either the linter does not apply in that particular situation or as temporary measure. 

And if I see it somewhere I do wonder *why* it was left. And in this case I would have no idea in a month or two. 
So there are better chances someone will come around this and wonder what is this `nolint:revive` about, instead of what would be the name of this argument we are not using here.

Reply:
Okay, I can buy it. Let me do a clean up.